client/.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Use js format instead of `json`. Add `root` property to avoid making eslint going to the parent folder